### ISSUE TITLE
Support external camera via OTG cable

### ISSUE THREAD

**Issue by MessiasLima** [begeekmyfriend/yasea#552]
Theres a way to stream from an endoscope camera like [that](https://www.amazon.com/BlueFire-Android-Endoscope-Phones-Function/dp/B013HZCYXK) instead device default camera?

**Comment by begeekmyfriend** [comment]
Of course, you may sample with raw image format in a standalone thread as long as you send date into [onGetRgbaFrame](https://github.com/begeekmyfriend/yasea/blob/master/library/src/main/java/net/ossrs/yasea/SrsCameraView.java#L232) or [onGetYuvNV21Frame]https://github.com/begeekmyfriend/yasea/blob/master/library/src/main/java/net/ossrs/yasea/SrsEncoder.java#L399) or [onGetArgbFrame](https://github.com/begeekmyfriend/yasea/blob/master/library/src/main/java/net/ossrs/yasea/SrsEncoder.java#L455)
